ansible-vault show how to decrypt strings (#2238)
ansible/ansible-documentation: docs/docsite/rst/vault_guide/vault_encrypting_content.rst

    ansible-vault decrypt foo.yml bar.yml baz.yml

.. _decrypting_strings:

Decrypting encrypted strings
^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> If you just want to check the contenbt of an encrypted string you can also view it by passing it via stdin:

Reviewer: ```suggestion
If you just want to check the content of an encrypted string you can also view it by passing it via stdin:
```

.. code-block:: bash

    echo -e '$ANSIBLE_VAULT;1.1;V2\neyJrZXkiOiAiZ0FBQUFBQm5UYzlPUVgzeUc5NFo3R2pzYVNMSXVsdXA3Z0paMmczNVRtS0NqMUcwMTVx\nSU1JVDlJZlRrSXBkVThmLXhKS00xZGl6X3F3YXZmWWUteGJWaHNZZXZNWl9hMWZvLVRYM3ZUZDRvaHRR\nWkhIdkJmZEZWNlBwVjhNVjJFT05QbDFwandaazAiLCAiY2lwaGVydGV4dCI6ICJnQUFBQUFCblRjOU9u\nWmM2dDh2VEN5c3NTQVlyV0hMclNEOFZfSGd2eEVHdERCdkJfakFpcUpaWWNTV19sR2hPY0VsWEVweS0z\nQ0NBcmJfdUdsUEt0NzJuSmFxVVVmRFIzdz09In0=' | ansible-vault decrypt

> or just have ansible prompt you for input by running (use Ctrl+D twice to end the input), just like when using ``encrypt_string``:

Reviewer: ```suggestion
Alternatively you can use the following command to have Ansible prompt you for input (use Ctrl+D twice to end the input), just like when using ``encrypt_string``:
```
